src/components/Dataset/ContactList.js

```diff
@@ -1,7 +1,20 @@
-import React, { Fragment } from 'react';
-import { withRouter } from 'react-router';
-import { List, Skeleton } from 'antd';
-import { FormattedRelative } from 'react-intl';
+import React  from 'react';
+import { withRouter } from 'react-router-dom';
+import { Button, List, Row, Skeleton } from 'antd';
+import { FormattedMessage, FormattedRelative } from 'react-intl';
+import { prettifyUserType } from '../../api/util/prettifiers';
+
+// TODO think about CSSinJS for styles
```

Comment:
I'm for using the react-jss approach instead of inline css. I can live with a tiny inline css property, but not as a general rule please.